install/helm/agones/values.yaml

```diff
@@ -94,6 +94,55 @@ agones:
       failureThreshold: 3
       timeoutSeconds: 1
     allocationBatchWaitTime: 500ms
+  extensions:
```

Comment:
I would add a comment to the top of the `extensions` message that this whole message is only relevant under the `SplitControllerAndExtensions` feature gate.

Reply:
Done. Added a comment above it.